src/handlers/album_list.rs

```diff
@@ -37,6 +37,7 @@ pub fn handler(key: Key, app: &mut App) {
         }
         Key::Ctrl('d') => app.get_current_user_saved_albums_next(),
         Key::Ctrl('u') => app.get_current_user_saved_albums_previous(),
+        Key::Char('D') => app.delete_current_user_saved_album(),
```

Comment:
It is hard thinking about what the key mapping should be, but given that this is context specific to the album_list, I think this is good.